### PR TITLE
Add bondary flux output 

### PR DESCRIPTION
**Pull request by totto82**
This PR also refactor the computation of flows and flores to a separate call.

### CODE REVIEW

**Comment by bska** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
This looks good for the most part.  Just a couple of minor nits.

File: opm/models/discretization/common/tpfalinearizer.hh [hunk 1]
```diff
@@ -636,6 +642,71 @@ public:
...
+            ADVectorBlock adres(0.0);
+            const unsigned globI = bdyInfo.cell;
+            const IntensiveQuantities& insideIntQuants = model_().intensiveQuantities(globI, /*timeIdx*/ 0);
+            LocalResidual::computeBoundaryFlux(adres, problem_(), bdyInfo.bcdata, insideIntQuants, globI);
+            adres *= bdyInfo.bcdata.faceArea;
+
+            if (enableFlows) {
+                for (unsigned phaseIdx = 0; phaseIdx < numEq; ++ phaseIdx) {
+                    if (adres[phaseIdx].value() == 0)
+                        continue;
```
[the remark this answers is not in the document]
Comment (totto82) [reply opm/models/discretization/common/tpfalinearizer.hh]: No reason. Just some leftover from debugging the parallel output issue solved by https://github.com/OPM/opm-simulators/pull/5043

**Comment by totto82** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**